docker-compose.yml

```
@@ -2,11 +2,20 @@ version: "3.8"
 
 services:
   map-sync:
-    image: ghcr.io/CivPlatform/map-sync
+    # FIXME: image repository cannot have capital letters
+    #image: ghcr.io/CivPlatform/map-sync
+    build: .
```

```suggestion
    image: ghcr.io/civplatform/map-sync
```
I had totally forgotten about capitalization